packages/api/src/lockfileSchema.ts

```diff
@@ -0,0 +1,84 @@
+import type { FromSchema } from '@readme/api-core/lib';
+
+const lockfileApiSchema = {
+  type: 'object',
+  required: ['createdAt', 'identifier', 'installerVersion', 'integrity'],
+  properties: {
+    createdAt: {
+      type: 'string',
+      format: 'date-time',
+      description: 'The date that this SDK was installed.',
+      examples: ['2023-10-19T20:35:39.268Z'],
+    },
+    identifier: {
+      type: 'string',
+      description:
+        "A unique identifier of the API. This'll be used to do imports on `@api/<identifier>` and also where the SDK code will be located in `.api/apis/<identifier>`",
+      examples: ['petstore'],
+    },
+    installerVersion: {
+      type: 'string',
+      description: 'The version of `api` that was used to install this SDK.',
+      examples: ['7.0.0'],
+    },
+    integrity: {
+      type: 'string',
+      description:
+        'An integrity hash that will be used to determine on `npx api update` calls if the API has changed since the SDK was last generated.',
+      examples: ['sha512-ld+djZk8uRWmzXC+JYla1PTBScg0NjP/8x9vOOKRW+DuJ3NNMRjrpfbY7T77Jgnc87dZZsU49robbQfYe3ukug=='],
```

Comment:
I'm not sure what the SHA I had in here was for but this one is the SHA for the Petstore spec.

```suggestion
      examples: ['sha512-otRF5TLMeDczSJlrmWLNDHLfmXg+C98oa/I/X2WWycwngh+a6WsbnjTbfwKGRU5DFbagOn2qX2SRvtBGOBRVGg=='],
```